src/h3lib/lib/geoCoord.c

```diff
@@ -332,3 +395,56 @@ int64_t H3_EXPORT(numHexagons)(int res) {
                                    569707381193162L};
     return nums[res];
 }
+
+double area_triangle(GeoCoord a, GeoCoord b, GeoCoord c) {
+    // Surface area of a spherical triangle given by three lat/lng points
+    // input: radians
+    // ouput: unit sphere area
+
+    double A, B, C, S, T, E;
+
+    A = H3_EXPORT(pointDistRads)(b, c);
+    B = H3_EXPORT(pointDistRads)(c, a);
+    C = H3_EXPORT(pointDistRads)(a, b);
+
+    S = (A + B + C) / 2;
+
+    A = (S - A) / 2;
+    B = (S - B) / 2;
+    C = (S - C) / 2;
+    S = S / 2;
+
+    T = sqrt(tan(S) * tan(A) * tan(B) * tan(C));
+    E = 4 * atan(T);
+
+    return E;
+}
+
+double H3_EXPORT(cellAreaRads2)(H3Index h) {
+    GeoCoord c;
+    GeoBoundary gb;
+    double A = 0.0;
+
+    H3_EXPORT(h3ToGeo)(h, &c);
+    H3_EXPORT(h3ToGeoBoundary)(h, &gb);
+
+    // can probably optimize this by re-using the shared edges
+    int N = gb.numVerts;
```

Comment:
Same nit, would just go with `numVerts` here.

Reply:
I generally agree that descriptive names are usually better, but I think it makes sense to break the rule here. Aliasing `gb.numVerts` and removing the `int` removes the visual noise in the loop, highlighting the math and the shared symbols. I think shorter names help with that, and don't incur much cost in this instance because their scope is small (only 3 lines).

Reply:
OK OK... updated :)